lib/core/canvas_buffer.dart

```diff
@@ -67,13 +69,13 @@ class CanvasBuffer {
   /// Sets the original terminal cursor offset for relative rendering.
   ///
   /// Typically set to the terminal’s current cursor position before rendering.
-  setTerminalOffset(int x, int y) {
-    cursorOriginalX = x;
-    cursorOriginalY = y;
+  setTerminalOffset(int column, int line) {
+    cursorOriginalColumn = column;
```

Comment:
For a separate refactor PR:

* Does the code become cleaner if you introduce a `TerminalPosition` class with a column and line? It would reduce all places with 2 parameters to one parameter.